searches/jump_search.php

```diff
@@ -0,0 +1,50 @@
+<?php 
+// PHP program to implement Jump Search
+  
+function jumpSearch($arr, $x, $n)
```

Comment:
Jump search was already added by another user [here](https://github.com/TheAlgorithms/PHP/pull/71).  Although they ended up closing that PR and creating a new one, they did add it first.